pages/iam/concepts.mdx

```diff
@@ -54,27 +54,29 @@ You are the [Owner](#owner) of the Organization that is created with your Scalew
 
 Similarly, you can invite other users to be Guests in your Organization. Whereas Owners have full rights and access to all resources and features in their Organization, Guests have only the rights and permissions given to them via [policies](#policy).
 
-<Lightbox src="scaleway-iam-owners-guests.webp" alt="" />
+<Message type="important">
```

Comment:
@ldecarvalho-doc I would add in this important message that Adding guest is impossible.
+ remove the paragraph above "similarly, you can invite..."